WORKSPACE

```diff
@@ -52,3 +52,20 @@ local_repository(
     name = "case4",
     path = "e2e/case4",
 )
+
+############################################
+# Example npm dependencies
+
+load("@aspect_rules_js//npm:npm_import.bzl", "npm_translate_lock")
+
+npm_translate_lock(
+    name = "npm",
+    npmrc = "//:.npmrc",
+    pnpm_lock = "//:pnpm-lock.yaml",
```

Comment:
If these deps are just for the examples, should we root it under the `examples` folder?

Reply:
The deps themselves are specified in the `examples/package.json`; just the pnpm workspace is rooted at the root of the repo